widget/entry.go

```diff
@@ -118,15 +118,30 @@ func (e *Entry) Bind(data binding.String) {
 	e.Unbind()
 	e.textSource = data
 
+	var convertErr error
+	e.Validator = func(string) error {
```

Comment:
In case the dev has already attached Validator we should call that first;
```go
v := e.Validator
e.Validator = func(text string) error {
    if v != nil {
        if err := v(text); err != nil {
            return err
        }
    }
    return convertErr
}
```

Reply:
Unbind presents a complication, if we reset the Validator we lose the binding validator and any previous one the dev set. If we don't reset it we could end up chaining binding validators :/

Maybe we flip it around and say devs should only set the Validator after binding?

Reply:
I don't think we can do that, as the developer could set e.Validator directly and break this.
I think the only solution would be to support multiple validations (internally?)..?

Reply:
As in instead of `Validator func(string)` it would be `Validators []func(string)`?

Reply:
I don't know. I think we discussed previously `NavigatorSet` or similar, that could contain multiple and check them all as it could implement the same interface...

Reply:
Assuming you meant `ValidatorSet`, that would be fine with me, but `Bind` would have to check if the entry's current validator type is `ValidatorSet`, then append the new validator. `Unbind` would also need to remove it.

Perhaps we're overthinking/overcomplicating this :S

Reply:
I wasn't imagining that Bind would check the type at all, it would set a `ValidatorSet` with itself and the `e.Validator` content. Then `Unbind` would unpack it.

I do think that we may be overthining because, as much as this could work, developers could still just stomp all over it because the field is designed to be publicly settable.
I wonder if the only safe solution is to have a second, private, validator field that is also checked and put the logic in Entry...
It's the same complication with `Check.OnChanged`.